### ISSUE TITLE
Is there one less training step?

### ISSUE THREAD

**Issue by zyxzyx45** [eric-yyjau/pytorch-superpoint#116]
I think that after labeling the false truth value, do you still need to continue to train the feature point extraction network MagicPoint for this label, and then train the loss of the descriptor, that is, Superpoint, why do you directly train Superpoint after labeling?

**Comment by zyxzyx45** [comment]
Then I found Train_model_subpixel this function, what is this function for? Did he continue to train MagicPoint based on false truths?
